src/utils/createHtmlContracts.js

```diff
@@ -33,7 +32,7 @@ export const createHtmlContracts = async (
         loadSheetDataCallback,
       }
 
-      const vars = await evalFunction(templateFunction)(query, emsData)
+      const vars = await evalFunction(templateFunction)(query, people)
```

Comment:
I think we should just pass `person` here, so that we can get rid of 
`  const _ = emsData.find((p) => p.jiraId === id)`, as that seems to me the purpose of https://github.com/vacuumlabs/contract-generator/issues/7. 
```suggestion
      const vars = await evalFunction(templateFunction)(query, person)
```

Reply:
Agreed. I did the same for loading google sheets. It doesn't make it faster (with every call of template function the whole sheet has to be loaded), but passing parameters is more consistent now.